app/views/devise/mailer/unlock_instructions.mjml

```diff
@@ -0,0 +1,13 @@
+<mj-section>
+  <mj-column>
+    <mj-text>
+      <p>Hello <%= @resource.email %>!</p>
+
+      <p>Your account has been locked due to an excessive number of unsuccessful sign in attempts.</p>
+
+      <p>Click the link below to unlock your account:</p>
+
+      <p><%= link_to "Unlock my account", unlock_url(@resource, unlock_token: @token) %></p>
+    </mj-text>
+  </mj-column>
+</mj-section>
```

Comment:
We don't currently lock folks out, but we've had this module enabled for a while, so I went ahead and updated this email as well.